tests/test_modeling_tf_common.py

```diff
@@ -380,65 +366,191 @@ def test_pt_tf_model_equivalence(self):
                 else:
                     pt_inputs_dict[name] = torch.from_numpy(key.numpy()).to(torch.long)
 
-            with torch.no_grad():
-                pto = pt_model(**pt_inputs_dict)
-            tfo = tf_model(self._prepare_for_class(inputs_dict, model_class), training=False)
+            return pt_inputs_dict
+
+        def check_outputs(tfo, pto, model_class, names):
```

Comment:
Nit: would prefer something more explicit here, like `tf_output` and `pt_output` to make the code easier to read.